src/TrueLayer/ApiClient.cs

```diff
@@ -234,17 +240,26 @@ private Task<HttpResponseMessage> SendJsonRequestAsync(
 #endif
             }
 
-            return SendRequestAsync(httpMethod, uri, idempotencyKey, accessToken, httpContent, signature, cancellationToken);
+            return SendRequestAsync(
+                httpMethod,
+                uri,
+                idempotencyKey,
+                accessToken,
+                httpContent,
+                signature,
+                customHeaders,
+                cancellationToken);
         }
 
         private Task<HttpResponseMessage> SendRequestAsync(
             HttpMethod httpMethod,
             Uri uri,
-            string? idempotencyKey,
-            string? accessToken,
-            HttpContent? httpContent,
-            string? signature,
-            CancellationToken cancellationToken)
+            string? idempotencyKey = null,
+            string? accessToken = null,
+            HttpContent? httpContent = null,
+            string? signature = null,
+            IDictionary<string, string>? customHeaders = null,
```

Comment:
On the Java lib we have a [custom class](https://github.com/TrueLayer/truelayer-java/blob/main/src/main/java/com/truelayer/java/http/entities/Headers.java#L15) for these custom headers, as we do not want(ed) to give too much freedom. I'm telling you this for context, not because I think we have to follow the same approach. Both have pros and cons I guess

Reply:
I thought about doing that but I just wanted to do it very quickly for now. Also the ApiClient class is private within the SDK so only "us" can pass the custom headers to it 😄 
Will revisit it in future.

Reply:
Ah cool, I did not realise that 👍